configs/orchestrator/README.md

```diff
@@ -1,22 +1,82 @@
 # Orchestrator
 
-This will install Lokomotive clusters.
+This chart is used to deploy the orchestrator application which can be used to deploy benchmarking clusters, run benchmarks on them, etc. This chart can be deployed on any Lokomotive cluster regardless or the region, the underlying architecture or cloud provider.
 
-## Install:
+## General Usage
+
+### Generate Ephemeral SSH Key Pair
+
+Generate SSH key pair, which will be used to install the benchmarking clusters:
 
 ```bash
-helm install --values=values-real.yaml --create-namespace --namespace orchestrator orchestrator .
+export KEY_PATH="${HOME}/.ssh/id_rsa.bc-${RANDOM}"
+ssh-keygen -t rsa -b 8192 -f "${KEY_PATH}" -N "" -C "<please provide email id>"
 ```
+### Install
 
-## Upgrade
+Run the following command to install the helm chart:
 
 ```bash
-helm upgrade --values=values-real.yaml --namespace orchestrator orchestrator .
+helm install \
+    --values=values-real.yaml \
+    --set-file runScript=run.sh \
+    --set-file sshKey.public="${KEY_PATH}".pub \
+    --set-file sshKey.private="${KEY_PATH}" \
+    --create-namespace \
+    --namespace orchestrator \
+    orchestrator .
 ```
 
-## Delete
+Copy the the [values.yaml](values.yaml) file and make changes as necessary and rename it to `values-real.yaml`.
+
+Update the path of the SSH keys as required.
+
+### Upgrade
+
+```bash
+helm upgrade \
+    --values=values-real.yaml \
+    --set-file runScript=run.sh \
+    --set-file sshKey.public=$HOME/.ssh/id_rsa."${KEY_NAME}".pub \
```

Comment:
This still needs to be updated to the other example.

Reply:
oops, my bad.